File: src/main/java/com/indiepost/dto/TimelineRequest.java
package com.indiepost.dto;

public class TimelineRequest {
    private long timepoint;

    private boolean after;

    private int size;

    public TimelineRequest() {
    }

    public long getTimepoint() {
        return timepoint;
    }

    public void setTimepoint(long timepoint) {
        this.timepoint = timepoint;
    }

    public boolean isAfter() {
        return after;
    }

    public void setAfter(boolean after) {
        this.after = after;
    }

    public int getSize() {
        return size;
    }

    public void setSize(int size) {
        this.size = size;
    }
}

File: src/main/kotlin/com/indiepost/dto/TimelineRequest.kt
package com.indiepost.dto

data class TimelineRequest(
        var timepoint: Long = 0,

        var isAfter: Boolean = false,

        var size: Int = 0

) {
    constructor(timepoint: Long, size: Int) : this() {
        this.timepoint = timepoint
        this.size = size
    }

}
